generative_data_prep/processors/article_tokenizer.py

```diff
@@ -116,13 +119,16 @@ def __call__(self, article: Optional[str]) -> List[TokenizedSequence]:
             return self.packer(article)
 
         tokenized_articles = []
-        try:
+        if self.file_ext == '.jsonl':
```

Comment:
Could you make the hardcoded strings '.jsonl' and '.txt' constants in an Enum in constants.py?

Reply:
Done!